release_notes/ocp-4-10-release-notes.adoc

```diff
@@ -255,6 +255,75 @@ For more information, see xref:../networking/ingress-operator.adoc#nw-configurin
+For more information, see xref:../storage/container_storage_interface/persistent-storage-csi-vsphere.adoc[vSphere CSI Driver Operator].
+
+[id="ocp-4-10-storage-azure-disk-csi-operator"]
+==== Persistent storage using Microsoft Azure Disk CSI Driver Operator is generally available
+
+{product-title} is capable of provisioning persistent volumes (PVs) using the Container Storage Interface (CSI) driver for Azure Disk. This feature was previously introduced as a Technology Preview feature in {product-title} 4.8 and is now generally available and enabled by default in {product-title} 4.10.
+
+For more information, see xref:../storage/container_storage_interface/persistent-storage-csi-azure.adoc[Azure Disk CSI Driver Operator].
+
+[id="ocp-4-10-storage-aws-efs-csi-operator"]
+==== Persistent storage using AWS Elastic File Storage CSI Driver Operator is generally available
+
+{product-title} is capable of provisioning persistent volumes (PVs) using the Container Storage Interface (CSI) driver for AWS Elastic File Storage (EFS). This feature was previously introduced as a Technology Preview feature in {product-title} 4.9 and is now generally available and enabled by default in {product-title} 4.10.
+
+For more information, see xref:../storage/container_storage_interface/persistent-storage-csi-aws-efs.adoc[AWS EFS Disk CSI Driver Operator].
+
+[id="ocp-4-10-storage-csi-migration-azure-file"]
+==== Automatic CSI migration supports Microsoft Azure file (Technology Preview)
+
+Starting with {product-title} 4.8, automatic migration for in-tree volume plug-ins to their equivalent CSI drivers became available as a Technology Preview feature. This feature now supports automatic migration for the Azure File in-tree plug-in to the Azure File CSI driver.
+
+For more information, see xref:../storage/container_storage_interface/persistent-storage-csi-migration.adoc[CSI automatic migration].
+
+[id="ocp-4-10-storage-csi-migration-vsphere"]
+==== Automatic CSI migration supports VMware vSphere (Technology Preview)
+
+Starting with {product-title} 4.8, automatic migration for in-tree volume plug-ins to their equivalent CSI drivers became available as a Technology Preview feature. This feature now supports automatic migration for the vSphere in-tree plug-in to the vSphere CSI driver.
+
+For more information, see xref:../storage/container_storage_interface/persistent-storage-csi-migration.adoc[CSI automatic migration].
+
+[id="ocp-4-10-storage-fsGroup-pod-timeouts"]
+==== Using fsGroup to reduce pod timeouts
+
+If a storage volume contains many files (~1,000,000 or greater), you may experience pod timeouts.
+
+{product-title} 4.10 introduces the ability to use `fsGroup` and `fsGroupChangePolicy` to skip recursive permission change for the storage volume, thus helping to avoid pod timeout problems.
+
+For more information, see xref:../storage/persistent_storage/understanding-persistent-storage#storage-persistent-storage-fsGroup.adoc[Using fsGroup to reduce pod timeouts].
```

Comment:
```suggestion
For more information, see xref:../storage/persistent_storage/understanding-persistent-storage.adoc#storage-persistent-storage-fsGroup[Using fsGroup to reduce pod timeouts].
```
With this one, you have an anchor, but I think your file extension is in the wrong place.